zenodo/base/upgrades/zenodo_2015_07_15_static_pages.py

```diff
@@ -102,7 +603,7 @@ <h4 id="restapi-res-">Quick start <small></small><span class="pull-right show-on
 </li>
 
 <li><p>Try to access the API:</p>
-<pre>>>> r = requests.get("{{config.CFG_SITE_SECURE_URL}}/api/deposit/depositions")
```

Comment:
We need to find a way to deal with this.

To have maintainable pages, we need to be able to use variables I think, otherwise this page will not work nicely on e.g. the sandbox (i.e. it will say https://zenodo.org instead of https://sandbox.zenodo.org).